Src/Plugins/Security/SEC101_001.HttpAuthorizationRequestHeaderValidator.cs

```diff
@@ -78,7 +78,14 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
             {
                 HttpClient client = CreateOrRetrieveCachedHttpClient();
                 using var requestDummy = new HttpRequestMessage(HttpMethod.Get, uri);
-                requestDummy.Headers.Authorization = new AuthenticationHeaderValue("Basic", ScanIdentityGuid);
+                if (options.ContainsKey("TestGuid"))
```

Comment:
>ContainsKey [](http://example.com/codeflow?start=28&length=11)

switch to trygetvalue, this will retrieve in one shot if that exists and if that contains a value